apps/admin/src/types.ts

```diff
@@ -89,7 +89,7 @@ export interface OrderItem {
   unitPrice: number;
   quantity: number;
   printSize: PrintSize;
-  volumes: number[];
+  volumes: [number, ...number[]];
```

Comment:
yup, it's really nice for avoiding typechecks that should not be needed. on the swift side, i use this library: https://github.com/pointfreeco/swift-nonempty which allows you to have collections (like arrays) that are statically guaranteed to never be empty. so i use those in swift, and i tweaked the codegen so that when i use one of those in swift, it generates this kind of type in Typescript.